pkg/relayer/proxy/server_builder.go

```diff
@@ -189,3 +104,27 @@ func (rp *relayerProxy) logRelayMinerConfiguredServices(supplierOperatorAddress
 	rp.logger.Info().Msgf("relayminer_configs for supplier %s: %v", supplierOperatorAddress, availableServices)
 
 }
+
+// logSupplierServices logs the services configured for a supplier.
+// It retrieves the supplier's onchain information and logs the services that the
+// supplier is configured to provide.
+func (rp *relayerProxy) logSupplierServices(ctx context.Context, supplierOperatorAddress string) {
+	supplier, err := rp.supplierQuerier.GetSupplier(ctx, supplierOperatorAddress)
+	if err != nil {
+		rp.logger.Error().Msgf(
+			"failed to get Supplier with address %q onchain information: %s",
+			supplierOperatorAddress,
+			err.Error(),
+		)
+	}
+
+	configuredServices := make([]string, 0)
+	for _, serviceConfig := range supplier.Services {
+		configuredServices = append(configuredServices, serviceConfig.ServiceId)
+	}
+	rp.logger.Info().Msgf(
+		"relayminer_configs for supplier %s: %v",
+		supplierOperatorAddress,
+		configuredServices,
```

Comment:
I think it might be easier if you log:

MsgF("[%s], string.sjoin(configuredServices, ","))